### PR TITLE
string cleaning funcs

### PR DESCRIPTION
**Pull request by jgibson517**
Let me know if this meets your needs, otherwise happy to re-write the function

### CODE REVIEW

File: civiclens/utils/text.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import re
+
+
+def clean_text(text: str, patterns: list[tuple] = []) -> str:
```
Comment (andrewjtdunn) [inline civiclens/utils/text.py:4]: is patterns intended to be actually used? If so, could use a definition in the documentation

**Review by andrewjtdunn — COMMENTED** [review]
Just the one comment about documentation

More broadly, which of these functions should we use to clean the summary text? Just the first?

**Review by GregoryHo88 — APPROVED** [review]
Thanks, this will be useful.

File: civiclens/utils/text.py (hunk 1 above)
Comment (jgibson517) [reply civiclens/utils/text.py:4]: sorry i left this out of the docstring, the three fixed regex statements were things I came across when processing text, and the patterns can allow someone else to input additional cleaning statements.

its not a perfect function, i'll admit

**Pull request merged**